### DISCUSSION TITLE
Status of PCI Passthrough on Single GPU

### DISCUSSION THREAD

**Discussion by freechelmi** [winapps-org/winapps#5] [General]
I'm quite new on PCI Passthrough and read of lot of different approach

AFAIK on the major case single GPU laptop or desktop  : 

- We can have native GPU perf on KVM using PCIPT
- We could using looking glass to use a single app running on VM on the Same GPU ? 
- does this fit the winapps use case ?

**Comment by LDprg** [comment]
How about this: https://github.com/BigAnteater/KVM-GPU-Passthrough

I also think Gpu Passthrough is definitively a usecase for winapps. Think about CAD Software (for example Fusion360, Solidworks, Inventor). They don't work (or not very good) with wine.

**Reply by oskardotglobal** [reply]
Yes, although it isn't a priority for now

**Reply by Boyfinn** [reply]
Solidworks user here.
Ever since the 2024 update for Solidworks connected, you have to either spoof or pass through some of your hardware info to QEMU/KVM. Both the hardware compatibility, and the Solidworks Connected app outright refuse to launch if the CPU is recognized as QEMU virtual PC or GPU as a RDP-session of any kind.

**Comment by oskardotglobal** [comment]
By the way, in case we have latency issues using RDP (which definitely exist and could ruin your day in cad software or especially in games) we could use https://github.com/LizardByte/Sunshine and https://github.com/moonlight-stream/moonlight-qt as a faster alternative (although its only really useful for games, since you can't just stream a single app, just the whole screen)

**Reply by fbartels** [reply]
Yes, maybe it is a good idea to have an additional alternative to rpd.

**Comment by LDprg** [comment]
We could also consider using https://github.com/HikariKnight/quickpassthrough as it nicely fits to quickemu for gpu passthrough.

**Reply by oskardotglobal** [reply]
Does that support single-gpu-passthrough though? Doesn't seem like it

**Reply by LDprg** [reply]
sadly not although it seems like it is planned

**Comment by freechelmi** [comment]
Not passthrough but it seems some new protocol "Venus" can help Virtio GPU performance

https://gist.github.com/peppergrayxyz/fdc9042760273d137dddd3e97034385f#file-qemu-vulkan-virtio-md

**Reply by nhandt64** [reply]
Venus does not support Windows guests.
